ptvsd/wrapper.py

```diff
@@ -418,7 +418,11 @@ def make_packet(self, cmd_id, args):
         with self.lock:
             seq = self.seq
             self.seq += 1
-        s = u'{}\t{}\t{}\n'.format(cmd_id, seq, args)
+        try:
+            s = u'{}\t{}\t{}\n'.format(cmd_id, seq, args)
+        except UnicodeDecodeError:
+            args = args.encode('ascii','replace')
```

Comment:
I don't think this is the right place to do it. The specific call that causes #845 - which is `pydevd_request(...)` in `_apply_code_stepping_settings` - should be the one ensuring that `args` is properly encoded, because only it knows the correct encoding to apply (and "ascii" with "replace" here will just mangle the argument).

In this particular case, since the caller is trying to pass in a path, it should check if it's `bytes` first, and if it is, *de*code it using `sys.getfilesystemencoding()`, so that it's always passing a proper unicode string to `send_request`.